app/views/console/users/user.phtml

```diff
@@ -93,6 +93,36 @@
                                 No user preferences found
                             </div>
                         </div>
+
+                        <h3 class="text-danger">Danger Zone</h3>
+
+                        <div class="box danger">
+                            <p>This is the area where you can delete this user.</p>
+                        
+                            <p>By deleting this user you will lose all data associated with this user.</p>
+                        
+                            <p>PLEASE NOTE: User deletion is irreversible.</p>
+                        
+                            <form class="inline"
+                                data-analytics-event="submit"
+                                data-analytics-category="console"
+                                data-analytics-label="Delete User"
+                                data-service="users.deleteUser"
+                                data-event="submit"
+                                data-param-user-id="{{router.params.id}}"
+                                data-success="alert,trigger,redirect"
+                                data-confirm="Are you sure you want to delete this user?"
+                                data-success-param-alert-text="Deleted user successfully"
+                                data-success-param-trigger-events="users.update"
```

Comment:
Probably missed that in the previous review. The event here should be `users.delete`

Reply:
Fixed it.